test/test_workspaces.py

```diff
@@ -41,3 +42,18 @@ def test_order_paths(self):
         self.assertEqual(['foo', 'bar'], order_paths(['bar', 'foo'], ['foo']))
         self.assertEqual(['baz', 'foo', 'bar'], order_paths(['bar', 'foo', 'baz'], ['baz', 'foo']))
         self.assertEqual(['foo' + os.sep + 'bim', 'bar'], order_paths(['bar', 'foo' + os.sep + 'bim'], ['foo']))
+
+    @unittest.skipUnless(sys.platform.startswith('linux'), 'requires Unix')
+    def test_order_paths_with_symlink(self):
+        try:
+            root_dir = tempfile.mkdtemp()
```

Comment:
This line should be outside of the `try` block.

Reply:
I aimed at consistency - previous two testcases made tmpdir within try block. I put the line before the try block - in previous tests as well.